tango_simlib/sim_xmi_parser.py

```diff
@@ -278,13 +294,20 @@ def extract_device_class_descr(self, description_data):
                                                 # information about the Tango device
                                                 # class, however it is not useful for
                                                 # the current problem.
+        #class_data["identification"] = {}
+        #identification = description_data.find('identification')
+        #class_data["identification"]["contact"] = identification.attrib["contact"]
+        #class_data["identification"]["author"] = identification.attrib["author"]
+        #class_data["identification"]["emailDomain"] = (
+        #    identification.attrib["emailDomain"])
+        #    class_data["identification"].append(id.attrib)
         class_data = {}
         class_data["super_classes"] = []
         super_classes = description_data.findall('inheritances')
         for super_class in super_classes:
             class_data["super_classes"].append(super_class.attrib)
 
-        return class_data
+        self.class_description.update(class_data)
```

Comment:
The docstring still promises to return a dict, so either do so, or update the docstring.